psydac/core/bsplines.py

```diff
@@ -108,6 +109,7 @@ def find_spans(knots, degree, x, out=None):
     spans : array of ints
         Knots span indexes.
     """
+    x = np.asarray(x)
```

Comment:
You might want to add `dtype=float`, because this will not work if an array of integers is given.

Reply:
Done in  54fa1fb